### ISSUE TITLE
Turn decorations On/Off automatically when Talon active/inactive

### ISSUE THREAD

**Issue by BoltonBailey** [cursorless-dev/cursorless#2017]
Hello, cool project, I am just trying it out. Not sure how easy this would be to achieve, but I figured I would post an enhancement request.

I noticed after turning off Talon that the decorations in my VSCode window remained. I think it could be nice if there could be a setting to turn the decorations on when and only when Talon is listening and Cursorless is ready to go, both so that the user has a visual indication of whether things are active and also so that the visual field is not cluttered if the user decides to turn Talon off for a while.

**Comment by pokey** [comment]
Interesting idea. @AndreasArvidsson i guess we could achieve this using our state file? This does seem like the kind of rapidly(-ish) changing thing that we might want to use a separate json for so we don't refresh spoken forms every time they sleep talon

**Comment by josharian** [comment]
Folks who want this could add the relevant toggle calls Talon-side.

**Comment by pokey** [comment]
Yeah that's true. Tho we should prob add commands to turn the hats on and off specifically in addition to the toggle to make that more robust
